esp/services/esdl_svc_engine/esdl_store.cpp

```diff
@@ -320,6 +458,15 @@ class CEsdlSDSStore : implements IEsdlStore, public CInterface
         else
             definitionInfo->setProp("@created",dt.getString(str).str());
 
+#ifdef ESDL_STORAGE_IS_STRING
```

Comment:
It was helpful during testing and I thought it could possibly be a option for a custom compilation of the platform if a user didn't want to use the new style. If you think it should be streamlined to unconditionally use the new style I'm fine with that.

Reply:
Considering that the purpose of this is to ensure order, and choosing not to use this guarantees that order is not maintained, I'm not sure it's something we should encourage. If there is a need for doing it, maybe the name could be changed to something that reflects the consequences of doing it.

When it comes to ensuring element order of a binding definition, I think bypassing this type of logic should not be supported. Reordering script elements could completely alter the script outcome.

Reply:
OK that makes sense. I'll remove prior to my final commit.